### PR TITLE
Adding crude html report support.

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

File: tools/run_tests/run_node.sh [hunk 1]
```diff
@@ -48,8 +48,8 @@ then
   cd build
   gcov Release/obj.target/grpc/ext/*.o
   lcov --base-directory . --directory . -c -o coverage.info
-  genhtml -o ../ext_coverage --num-spaces 2 -t 'Node gRPC test coverage' \
-    coverage.info
+  genhtml -o ../../../reports/node_ext_coverage --num-spaces 2 \
+    -t 'Node gRPC test coverage' coverage.info
```
Comment (murgatroid99) [inline tools/run_tests/run_node.sh]: If you're changing this, you might also want to change lines 46 and 47 to

```sh
./node_modules/.bin/istanbul cover --dir ../../reports/node_coverage \
  ./node_modules/.bin/_mocha -- --timeout 8000
```

Comment (nicolasnoble) [inline tools/run_tests/run_node.sh]: Thanks :-) I'm merely doing some Jenkins tests for now, but I'll change that too.

**Comment by murgatroid99** [comment]
It looks good, but I would prefer to hold off on merging until the sanity tests are fixed. That yellow circle is making me nervous.

**Comment by nicolasnoble** [comment]
Fixed in #3611.

**Pull request merged**